samples/snippets/src/test/java/com/google/cloud/examples/securitycenter/snippets/MuteFindingIT.java

```diff
@@ -0,0 +1,234 @@
+  }
+
+  @AfterClass
+  public static void cleanUp() throws IOException {
+    final PrintStream out = System.out;
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+    DeleteMuteRule.deleteMuteRule(PROJECT_ID, MUTE_RULE_CREATE);
+    DeleteMuteRule.deleteMuteRule(PROJECT_ID, MUTE_RULE_UPDATE);
+    stdOut = null;
+    System.setOut(out);
+  }
+
+  public static Source createSource(String organizationId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Source source =
+          Source.newBuilder()
+              .setDisplayName("Custom display name")
+              .setDescription("A source that does X")
+              .build();
+
+      CreateSourceRequest createSourceRequest =
+          CreateSourceRequest.newBuilder()
+              .setParent(String.format("organizations/%s", organizationId))
+              .setSource(source)
+              .build();
+
+      Source response = client.createSource(createSourceRequest);
+      System.out.println("Created source : " + response.getName());
+      return response;
+    }
+  }
+
+  public static Finding createFinding(String sourceName, String findingId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      Instant eventTime = Instant.now();
+
+      // The resource this finding applies to.  The Cloud Security Command Center UI can link
+      // the findings for a resource to the corresponding Asset of a resource
+      // if there are matches.
+      // TODO(Developer): Replace the below sample resource name
+      String resourceName = "//cloudresourcemanager.googleapis.com/organizations/11232";
+
+      // Start setting up a request to create a finding in a source.
+      Finding finding =
+          Finding.newBuilder()
+              .setParent(sourceName)
+              .setState(State.ACTIVE)
+              .setSeverity(Severity.LOW)
+              .setMute(Mute.UNMUTED)
+              .setFindingClass(FindingClass.OBSERVATION)
+              .setResourceName(resourceName)
+              .setEventTime(
+                  Timestamp.newBuilder()
+                      .setSeconds(eventTime.getEpochSecond())
+                      .setNanos(eventTime.getNano()))
+              .setCategory("LOW_RISK_ONE")
+              .build();
+
+      Finding response = client.createFinding(sourceName, findingId, finding);
+
+      System.out.println("Created Finding: " + response);
+      return response;
+    }
+  }
+
+  public static ListFindingsPagedResponse getAllFindings(String sourceName) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      ListFindingsRequest request = ListFindingsRequest.newBuilder().setParent(sourceName).build();
+
+      return client.listFindings(request);
+    }
+  }
+
+  private static String getOrganizationId() {
+    return "1081635000895";
```

Comment:
It's better to save this as an env var and store it in java-docs-samples-testing project's secret manager.